README.md

```diff
@@ -1,19 +1,23 @@
-## zbMATH links API
+## zbMATH Links API
 ![test status](https://github.com/zbmathopen/linksApi/actions/workflows/pytest.yml/badge.svg)
 
-1) Install requirements and set the env variables.
+The purpose of the zbMATH Links API is to show the interconnections between [zbMATH](https://zbmath.org/) (the `target` of the API)
+and external platforms (called here `partners` and constituting the `source` of the API) which store links linking to objects in the target, i.e.,
+documents indexed at zbMATH. The prototypical partner is the [Digital Library of Mathematical Functions](https://dlmf.nist.gov/) (DLMF), which
+contains more than 6.000 links linking to publications indexed at zbMATH. Other partners can be integrated as well.
+
+To run the zbMATH Links API with a dummy database (only for illustrative purposes) please follow these steps:
+
+1) Install the requirements and set the environment variables.  On a first install:
```

Comment:
suggest starting each sentence in a new line. (Like in LaTeX one new line does not create a new paragraph in GitHub flavored markdown.)

Reply:
I pushed the changes